source/SquadLeader/src/com/esri/squadleader/controller/AdvancedSymbolController.java

```diff
@@ -169,13 +203,27 @@ protected boolean processMessage(Geomessage geomessage) {
             message.setID(geomessage.getId());
         }
         
+        return _processMessage(message);
+    }
+    
+    private boolean _processMessage(Message message) {        
+        /**
+         * Workaround: ArcGIS Runtime 10.2.4 requires a chem light message to have
```

Comment:
Yes, Esri Support has logged a bug for this behavior. I have documented this bug in f9f8fe800201602daac832d8e75c78edef48f9fe .